command/bridge/common/params.go

```diff
@@ -0,0 +1,78 @@
+package common
+
+import (
+	"errors"
+)
+
+type TokenType int
+
+const (
+	ERC20 TokenType = iota
+	ERC721
+	ERC1155
+)
+
+const (
+	SenderKeyFlag = "sender-key"
+	ReceiversFlag = "receivers"
+	AmountsFlag   = "amounts"
+	TokenIDsFlag  = "token-ids"
+
+	RootTokenFlag      = "root-token"
+	RootPredicateFlag  = "root-predicate"
+	ChildPredicateFlag = "child-predicate"
+	ChildTokenFlag     = "child-token"
+	JSONRPCFlag        = "json-rpc"
+)
+
+var (
+	errInconsistentAmounts  = errors.New("receivers and amounts must be equal length")
+	errInconsistentTokenIds = errors.New("receivers and token ids must be equal length")
+)
+
+type BridgeParams struct {
+	SenderKey     string
+	Receivers     []string
+	TokenAddr     string
+	PredicateAddr string
+	JSONRPCAddr   string
+}
+
+type ERC20BridgeParams struct {
+	*BridgeParams
+	Amounts []string
```

Comment:
Could we move this `Amounts` slice to `BridgeParams` struct, since you use it in both `ERC20BridgeParams` and in `ERC1155BridgeParams`?

Reply:
It would be misleading for ERC721 tokens, because they don't have it. So I'd still prefer to leave it as is.